pages/edit.js

```diff
@@ -0,0 +1,365 @@
+                  value={newTimer.name}
+                  placeholder={`Exercise ${newTimer.id}`}
+                  onChange={(event) =>
+                    changeTimerChain(
+                      event,
+                      newTimer.id,
+                      newTimerChain,
+                      setNewTimerChain
+                    )
+                  }
+                />
+                <Label htmlFor="name"> Exercise</Label>
+              </InputAndLabelWrapper>
+              <InputAndLabelWrapper>
+                <Select
+                  id="minutes"
+                  value={newTimer.minutes}
+                  onChange={(event) =>
+                    changeTimerChain(
+                      event,
+                      newTimer.id,
+                      newTimerChain,
+                      setNewTimerChain
+                    )
+                  }
+                >
+                  {timeOptions.map((option) => {
+                    return (
+                      <option key={option} value={option}>
+                        {option}
+                      </option>
+                    );
+                  })}
+                </Select>
+                <Label htmlFor="minutes">min.</Label>
+              </InputAndLabelWrapper>
+              <span>:</span>
+              <InputAndLabelWrapper>
+                <Select
+                  id="seconds"
+                  value={newTimer.seconds}
+                  onChange={(event) =>
+                    changeTimerChain(
+                      event,
+                      newTimer.id,
+                      newTimerChain,
+                      setNewTimerChain
+                    )
+                  }
+                >
+                  {timeOptions.map((option) => {
+                    return (
+                      <option key={option} value={option}>
+                        {option < 10 ? `0${option}` : option}
+                      </option>
+                    );
+                  })}
+                </Select>
+                <Label htmlFor="seconds"> sec.</Label>
+              </InputAndLabelWrapper>
+            </Li>
+          ))}
+        </FormList>
+        <ButtonWrapper>
+          <AddButton
+            type="button"
+            onClick={() => handleAdd(newTimerChain, setNewTimerChain)}
+          >
+            <Image alt="add" src={add} width="64" height="64" />
+          </AddButton>
+          <DeleteButton
+            type="button"
+            onClick={() => handleRemove(newTimerChain, setNewTimerChain)}
+          >
+            <Image alt="remove" src={remove} width="64" height="64" />
+          </DeleteButton>
+        </ButtonWrapper>
+        <ApplyWrapper>
+          <DummyDiv />
+          <Apply type="submit">
+            <Image alt="hook" src={hook} width="42" height={41.71} />
+            <div>Apply</div>
+          </Apply>
+        </ApplyWrapper>
+      </form>
+      <DownGap />
```

Comment:
Why not make the `<form>` a styled component with a padding-bottom: 50px instead of having a new styled component `<DownGap />`?